requirements.txt

```diff
@@ -4,6 +4,6 @@ srsly>=2.4.0,<3.0.0
 # Development requirements
 pathy>=0.3.5
 pytest>=5.2.0,!=7.1.0
-mypy>=0.901,<0.970; platform_machine!='aarch64'
+mypy>=0.980,<0.990; platform_machine!='aarch64' and python_version > "3.6"
```

Comment:
I'm trying to decide how much I care about the mixed quotes, spacing, and `>`/`<`...

Reply:
Eh, I'll just change it.